lib/runtime/wasmer/test_helpers.go

```diff
@@ -45,6 +46,17 @@ func NewTestInstanceWithTrie(t *testing.T, targetRuntime string, tt *trie.Trie)
 	return r
 }
 
+func mustHexTo64BArray(t *testing.T, inputHex string) (outputArray [64]byte) {
```

Comment:
Since this function is defined twice and seems rather general purpose, I think we can add this in lib/common, you can move this function there and introduce common.Signature

`type Signature [64]byte`